Makefile

```diff
@@ -97,8 +97,8 @@ ifdef NO_DOCKER
 	scBuildImageTarget =
 else
 	# Mount .pkg as pkg so that we save our cached "go build" output files
-	DOCKER_CMD = docker run --rm -v $(PWD):/go/src/$(SC_PKG) \
-	  -v $(PWD)/.pkg:/go/pkg scbuildimage
+	DOCKER_CMD = docker run --rm -v $(PWD):/go/src/$(SC_PKG):z \
```

Comment:
I am concerned that the :z flag might not be the best approach to solving the permissions issue. There are some known downsides to allowing docker to relabel the volumes. Another approach would be to use `--priveleged` or `--cap-add`.

Reply:
There are concerns raising in [this moby issue](https://github.com/moby/moby/issues/30934) that the :z flag may have undesirable side effects on the host.

Reply:
What an interesting case, I didn't consider trying to execute the makefile from anywhere else other than within the git checkout. Maybe replace $(PWD) with $(ROOT) and then the danger of relabeling the wrong files goes away?